src/flushMicrotasksQueue.js

```diff
@@ -1,7 +1,15 @@
 // @flow
+
+import { printDeprecationWarning } from './helpers/errors';
+
 /**
  * Wait for microtasks queue to flush
  */
 export default function flushMicrotasksQueue(): Promise<any> {
+  printDeprecationWarning('flushMicrotasksQueue');
+  return flushMicrotasksQueueInternal();
+}
+
+export function flushMicrotasksQueueInternal(): Promise<any> {
```

Comment:
Can we call it just `flushMicrotasks`?

Reply:
Renamed to `flushMicroTasks` as this is how it's spelled in RTL